### ISSUE TITLE
Error preventing bundling Meteor with Ionic 3 and Angular2

### ISSUE THREAD

**Issue by laurentenhoor** [Urigo/angular-meteor#1883]
```bash
Errors prevented bundling:
While loading plugin `basicFileTypes` from package `meteor`:
packages/basicFileTypes_plugin.js:39:9: Package._define is not a function
at packages/basicFileTypes_plugin.js:39:9
at packages/basicFileTypes_plugin.js:41:3


/usr/local/lib/node_modules/meteor-client-bundler/node_modules/execa/index.js:277
                throw (result.error || new Error(result.stderr === '' ? result.stdout : result.stderr));
                ^

Error: null
    at Function.module.exports.sync (/usr/local/lib/node_modules/meteor-client-bundler/node_modules/execa/index.js:277:26)
    at Object.bundle (/usr/local/lib/node_modules/meteor-client-bundler/lib/bundler.js:97:9)
    at Command.<anonymous> (/usr/local/lib/node_modules/meteor-client-bundler/cli:47:13)
    at Command.listener (/usr/local/lib/node_modules/meteor-client-bundler/node_modules/commander/index.js:315:8)
    at emitTwo (events.js:125:13)
    at Command.emit (events.js:213:7)
    at Command.parseArgs (/usr/local/lib/node_modules/meteor-client-bundler/node_modules/commander/index.js:651:12)
    at Command.parse (/usr/local/lib/node_modules/meteor-client-bundler/node_modules/commander/index.js:474:21)
    at Object.<anonymous> (/usr/local/lib/node_modules/meteor-client-bundler/cli:55:9)
    at Module._compile (module.js:624:30)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! my_project@0.0.1 meteor-client:bundle: `meteor-client bundle -s api`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the my_project@0.0.1 meteor-client:bundle script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by hongan126** [comment]
Yesterday it was working.
Today, I have the same proplem.
So I recovery my Windows on time with no errors.

**Try reinstall meteor.
And check plugin basicFileTypes in the path:**
>C:\Users\<your-users-name>\AppData\Local\.meteor\packages\meteor\<version>\plugin.basicFileTypes.os

![scr_1-29-2018-9 49 18 am2-25-2018-11 58 30 am](https://user-images.githubusercontent.com/25642719/36638320-805a29e2-1a24-11e8-89b0-eba4419e3dde.png)

cmd:
```
> ftel-notes-angularcli-meteor@0.0.1 meteor-client:bundle D:\ftel-notes-angularcli-meteor
> meteor-client bundle -s api

Created a new Meteor app in 'C:\Users\honga\AppData\Local\Temp\tmp-4816mElo4zNH33LD'.

To run your new app:
  cd C:\Users\honga\AppData\Local\Temp\tmp-4816mElo4zNH33LD
  meteor

If you are new to Meteor, try some of the learning resources here:
  https://www.meteor.com/tutorials

meteor create --bare to create an empty app.
meteor create --full to create a scaffolded app.

up to date in 2.426s

Changes to your project's package version selections:

accounts-base                 added, version 1.4.2
accounts-password             added, version 1.5.0
barbatus:typescript           added, version 0.6.12
barbatus:typescript-compiler  added, version 0.9.12
barbatus:typescript-runtime   added, version 1.0.2
ddp-rate-limiter              added, version 1.0.7
email                         added, version 1.2.3
localstorage                  added, version 1.2.0
npm-bcrypt                    added, version 0.9.3
rate-limit                    added, version 1.0.8
service-configuration         added, version 1.0.11
sha                           added, version 1.0.9
srp                           added, version 1.0.10

WARNING: The output directory is under your source tree.
         Your generated files may get interpreted as source code!
         Consider building into a different directory instead
         meteor build ../output

   ╭─────────────────────────────────────╮
   │                                     │
   │   Update available 5.5.1 → 5.6.0    │
   │     Run npm i -g npm to update      │
   │                                     │
   ╰─────────────────────────────────────╯
```

**Comment by laurentenhoor** [comment]
Thanks. Indeed fully reinstalling Meteor works.
My first attempt was deleting the ~/.meteor folder manually and run the installation command afterwards, but this was not sufficient to fix this issue.

So re-install by running the following command works, it deletes the older version automatically.
```bash
curl https://install.meteor.com/ | sh
```
